wp-includes/sqlite/class-wp-sqlite-translator.php

```diff
@@ -4188,4 +4192,18 @@ public function rollback() {
 		do_action( 'sqlite_transaction_query_executed', 'ROLLBACK', (bool) $this->last_exec_returned, $this->transaction_level );
 		return $this->last_exec_returned;
 	}
+
+	/**
+	 * Create an index name consisting of table name and original index name.
+	 * This is to avoid duplicate index names in SQLite.
+	 *
+	 * @param $table
+	 * @param $original_index_name
+	 *
+	 * @return string
+	 */
+	private function generate_index_name( $table, $original_index_name ) {
+		// Strip the occurrences of 2 or more consecutive underscores to allow easier splitting on __ later.
```

Comment:
@jeroenpf it looks like this would make it so we cannot derive the original mysql index name from sqlite index name because we cannot tell which underscores used to be double-underscores.

So import/export operations are not symmetrical/reversible. Do we care about that? Offhand, I think maybe we should.

Reply:
@brandonpayton for this reason i strip out double underscores from the table name alone. Later we can split on the first occurrence of double underscores. The idea was that it does not matter if the index name itself has double underscores so they would be preserved. Maybe I am overlooking something?

Example:

`table__name` and `some_index__name` becomes `table_name__some_index__name` and later we split on the first occurrence of `__` which which leaves us with [`table_name`, `some_index__name`].

I do however see one error in the explode logic that is used to get the index name; i should have passed in a limit so that it only separates on the first occurrence of `__`.  I will create a new PR to address this.

Reply:
> for this reason i strip out double underscores from the table name alone.

@jeroenpf thank you for pointing that out. I had missed that this was just for the separator between table name and index. The naming scheme makes sense to me now.